docs/api/index.html

```diff
@@ -4443,6 +4497,17 @@ <h3 id="planetscale.com/v2.VitessImages">VitessImages
 </tr>
 <tr>
 <td>
+<code>vtadmin</code></br>
+<em>
+string
+</em>
+</td>
+<td>
+<p>Vtadmin is the container image (including version tag) to use for Vtadmin instances.</p>
```

Comment:
```suggestion
<p>Vtadmin is the container image (including version tag) to use for Vitess Admin instances.</p>
```

Reply:
Not quite sure what this should _really_ be called, so this is just a suggestion.

Reply:
Is this optional? My recollection is that for other images (like vttablet), we default to `vitess/lite:latest` if the image is not specified. But since vtadmin is still experimental, if this image is not provided we should skip the deployment.

Reply:
Fixed. Also added a check in the code for the same. If the image is not specified, we log an error and don't deploy vtadmin